mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -871,7 +871,8 @@ FailureOr<VectorizationResult>
 vectorize(RewriterBase &rewriter, Operation *op,
           ArrayRef<int64_t> inputVectorSizes = {},
           ArrayRef<bool> inputScalableVecDims = {},
-          bool vectorizeNDExtract = false, bool flatten1DDepthwiseConv = false);
+          bool vectorizeNDExtract = false, bool flatten1DDepthwiseConv = false,
+          bool assumeScalableSizesMultipleOfDim = false);
```

Comment:
We have `assumeScalableSizesMultipleOfDim` and `getAssumeScalableSizesMatchDimSize`. Should we have only one?
Also, it should be "dim multiple of vector sizes"?

Reply:
Thanks for the suggestion, I went with `assumeDynamicDimsMatchVecSizes`, see this [commit](https://github.com/llvm/llvm-project/pull/146531/commits/2b6019caf04f37715ca5857b31b3f4fe9c48a417). As mentioned in my other comment, for now I am "assuming" **equality** rather than **divisibility**.  Not sure whether we will need the latter?